phpstan.neon

```diff
@@ -20,6 +20,7 @@ parameters:
 		- src/Persistence/Search/SearchIndexFieldsBuilder.php
 		- src/Persistence/Search/Query/HasWbFacetFeature.php
 		- src/Persistence/Elastic*.php
+		- src/WikibaseFacetedSearchExtension.php
```

Comment:
This file got contaminated by CirrusSearch classes: https://github.com/ProfessionalWiki/WikibaseFacetedSearch/pull/44#discussion_r1888899011